ab_initio_calculations/utils/chemical_utils.py

```diff
@@ -22,14 +55,34 @@ def guess_metal(ase_obj) -> bool:
     returns bool
     """
     non_metallic_atoms = {
```

Comment:
@alinzh I'd recommend to keep this layout with spaces, because then one can see a part of the periodic table here, explaining clearly why these elements bear non-metallic character